recap/recap.py

```diff
@@ -589,6 +589,11 @@ def make_pdf_json(self, data, suffix=''):
         blocks = self.get_blocks_list(user, which_blocks)
         html = self.get_user_layout(blocks, user)
 
+        if "<h3>" in html:
+            html = re.sub("<h3>(.*?)<\/h3>","<h3>{}</h3>".format(data['document_heading']), html)
```

Comment:
Does this also need the document heading/html wrapped in unicode?

Reply:
Luckily the RegEx parser handles unicode like a boss.